Графика для всех страниц

kattackkattack opened the pull request:
2 ДЗ. Разметка всех страниц
3 ДЗ. Графика для всех страниц

---
:mortar_board: [Графика для всех страниц](https://up.htmlacademy.ru/htmlcss/38/user/2355239/tasks/7)

kattackkattack committed:
Добавляет catalog.html

kattackkattack committed:
Добавляет базовую структуру документа

kattackkattack committed:
Добавляет шапку

Добавляет в catalog.html разметку шапки

kattackkattack committed:
Добавляет шапку в index.html

kattackkattack committed:
Добавляет секцию welcome

Добавляет первую секцию главной страницы

kattackkattack committed:
Добавляет секцию с преимуществами

kattackkattack committed:
Добавляет блоки с поиском по гостиницам и подпиской

kattackkattack committed:
Добавляет футер

kattackkattack committed:
Добавляет блок с фильтром

catalog.html — добавляет блок с фильтрами
index.html — мелкие правки

kattackkattack committed:
Добавляет счетчик гостиниц, сортировку и изменение превью

kattackkattack committed:
Добавляет каталог, пагинацию и блок с подпиской на рассылку

kattackkattack committed:
Добавляет обертки для преимуществ

kattackkattack committed:
Меняет настройки editor config

В соответствии с правилами HTML Academy

kattackkattack committed:
Изменяет форматирование каталога

kattackkattack committed:
Изменяет форматирование текста в head

kattackkattack committed:
Правит форматирование

kattackkattack committed:
Добавляет экшн для форм

kattackkattack committed:
Добавляет изображения

kattackkattack commented:
Добавила коммиты с третьим дз

kattackkattack committed:
Добавляет файл стилей

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+            </a>
+          </li>
+        </ul>
+        <button type="button">Хочу сюда!</button>
```
А почему вынесла кнопку из списка?

kattackkattack replied:
В учебных материалах говорили, что списки для однотипных элементов. По этой логике кнопка в список иконок не попадала

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+        <ul class="advantages-list">
+          <li class="advantages-item">
+            <section class="advantages-item-wrapper ligh-grey">
+              <img class="advantages-item-icon" src="#" alt="Домик">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+          </li>
+          <li class="advantages-item">
+            <section class="advantages-item-wrapper white">
+              <img class="advantages-item-icon" src="#" alt="Бургер">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+          </li>
+          <li class="advantages-item">
+            <section class="advantages-item-wrapper ligh-grey">
+              <img class="advantages-item-icon" src="#" alt="Коробка с подарком">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Избранное</span>
+              <img src="#" alt="Иконка избранного">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">
+              <span class="visually-hidden">Поиск</span>
+              <img src="#" alt="Иконка поиска">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+      <ul class="social-list">
+        <li class="social-item">
+          <a class="social-button" href="https://vk.com/htmlacademy" target="_blank">
+            <img class="social-icon" src="#" alt="Иконка VK">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+        </li>
+        <li class="social-item">
+          <a class="social-button" href="https://www.youtube.com/user/htmlacademyru" target="_blank">
+            <img class="social-icon" src="#" alt="Иконка YouTube">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+        </li>
+        <li class="social-item">
+          <a class="social-button" href="https://t.me/htmlacademy" target="_blank">
+            <img class="social-icon" src="#" alt="Иконка Telegram">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+          <li class="navigation-item">
+            <a class="navigation-link" href="#">
+              <span class="visually-hidden">Избранное</span>
+              <img src="#" alt="Иконка избранного">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+          <li class="navigation-item">
+            <a class="navigation-link" href="catalog.html">
+              <span class="visually-hidden">Поиск</span>
+              <img src="#" alt="Иконка поиска">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+            <ul class="breadcrumbs">
+              <li class="breadcrumbs-item">
+                <a class="breadcrumbs-link" href="index.html">
+                  <img class="breadcrumbs-icon" src="#" alt="Иконка домика">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+                </a>
+              </li>
+              <li>
+                <img class="breadcrumbs-arrow" src="#" alt="Стрелка вправо">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+
+            <!--filter-submit-->
+
+            <button type="submit">Применить</button>
```
Лучше добавить обертку для кнопок, будет легче стилизовать.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+            <li class="catalog-view-item">
+              <a class="catalog-view-link catalog-view-link-current" href="#">
+                <span class="visually-hidden">Отображение в виде сетки</span>
+                <img src="#" alt="Иконка вида сетки">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+            <li class="catalog-view-item">
+              <a class="catalog-view-link" href="#">
+                <span class="visually-hidden">Отображение в виде крупных превью</span>
+                <img src="#" alt="Иконка вида крупного превью">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+            <li class="catalog-view-item">
+              <a class="catalog-view-link" href="#">
+                <span class="visually-hidden">Отображение в виде списка</span>
+                <img src="#" alt="Иконка вида списка">
```
Это будет декоративное изображение, которое мы добавляем через CSS.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+            </section>
+            <section class="rating">
+              <div class="rating-star-set">
+                <img class="rating-star" src="#" alt="Иконка звезды">
```
Это будет декоративное изображение, которое мы добавляем через CSS.
И лучше сделать звезды через список, где каждая звезда это li и через класс например .active мы добавляем звездочку, если класса нет, то и звездочку не добавляем.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+              <p class="product-type">Гостиница</p>
+              <p class="product-cost">От 4000&#8201;₽</p>
+            </div>
+            <section class="product-item-actions">
```
Тег `section` больше подходит для больших блоков, в данном случае лучше использовать `div`.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+    <!--page-footer-->
+
+    <footer class="page-footer">
+      <h2 class="visually-hidden">Контакты разработчика</h2>
```
Это лишний элемент в принципе.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+
+    <footer class="page-footer">
+      <h2 class="visually-hidden">Контакты разработчика</h2>
+      <h3 class="visually-hidden">Социальные сети</h3>
```
Это лишний элемент в принципе.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+          </a>
+        </li>
+      </ul>
+      <h3 class="visually-hidden">Телефон</h3>
```
Это лишний элемент в принципе.

vlad-yermolayev commented on index.html:
```diff
@@ -3,10 +3,199 @@
+      </ul>
+      <h3 class="visually-hidden">Телефон</h3>
+      <a class="footer-phone" href="tel:+78128121212">+7 (812) 812-12-12</a>
+      <h3 class="visually-hidden">Сайт разработчика</h3>
```
Это лишний элемент в принципе.

vlad-yermolayev commented on catalog.html:
```diff
@@ -0,0 +1,350 @@
+            <!--filter-submit-->
+
+            <button type="submit">Применить</button>
+            <button type="button">Сбросить</button>
```
Тут должен быть `type="reset"`

kattackkattack committed:
Перемещает кнопку "Хочу сюда"

Перемещает кнопку внутрь списка, где лежат иконки

kattackkattack committed:
Добавляет фавикон

kattackkattack committed:
Добавляет Colors.css

Добавляет файл с переменными цвета

kattackkattack committed:
Подключает в файлы стили с переменными цвета

kattackkattack committed:
Подключает файл css со шрифтами

kattackkattack committed:
Заменяет иконки в спрайте

Предыдущие были с некорректными размерами

kattackkattack committed:
Переносит кнопку шапки в список с иконками

kattackkattack committed:
Добавляет файл со шрифтами

kattackkattack committed:
Исправляет путь к файлам со шрифтами

kattackkattack committed:
Исправляет разметку по комментариям наставника

— Заменила иконки на svg
— Добавила файл спрайта
— Убрала ненужные скрытые заголовки
— Изменила классы иконок в соответствии со стайлгайдом (не было в комментариях)
— Рейтинг исправила на список из звезд
— Обернула кнопки формы в блок
— Исправила секции на дивки

kattackkattack committed:
Добавляет изображения

kattackkattack committed:
Добавляет изображения

Изображения как img
Изображения как background-image у блоков

kattackkattack commented:
По дз #2 вроде все исправила

kattackkattack committed:
Добавляет общий контейнер для контента catalog.html

kattackkattack committed:
Добавляет токены типографики

kattackkattack committed:
Добавляет семейство к стилям шрифтов

kattackkattack committed:
Добавляет размеры иконок

kattackkattack committed:
Изменяет стили списков, ссылок, изображений

kattackkattack committed:
Добавляет размеры иконкам в catalog.html

kattackkattack commented:
Исправила остатки. Теперь с картинками и иконками должно быть все окей

kattackkattack committed:
Исправляет ошибку в размерах иконки подарка

kattackkattack committed:
Заменяет некоторые секции на дивы

kattackkattack committed:
Добавляет стилизацию

kattackkattack commented:
Ссылки в шапке я поправлю

kattackkattack committed:
Добавляет стили хедера и кнопок

kattackkattack committed:
Стилизует хедер

kattackkattack committed:
Стилизует главную страницу

Добавляет сетки блокам
Добавляет стили заголовкам и подписям
Добавляет стили блокам с преимуществами

kattackkattack committed:
Убирает разделитель в преимуществах с иконками

kattackkattack committed:
Исправляет ширину текста в блоке с подпиской

kattackkattack committed:
Добавляет box-sizing страницам

Не знаю куда подевался, уже указывала его

kattackkattack committed:
Растягивает изображения грида на 2 колонки

kattackkattack committed:
Стилизует фильтры и выдачу

kattackkattack committed:
Стилизует переключатель вида сетки

kattackkattack committed:
Причесывает цветовую палитру

kattackkattack committed:
Стилизует пагинацию, обновляет токены цветов

kattackkattack committed:
Стилизует хлебные крошки

kattackkattack committed:
Исправляет ошибки в HTML

+ исправляет иерархию заголовков

kattackkattack committed:
Добавляет имя поля email

kattackkattack committed:
Переносит стили breadcrumbs в файл для стилей контролов

kattackkattack committed:
Добавляет маску для hero

kattackkattack committed:
Заменяет контентные изображения на декоративные

kattackkattack committed:
Добавляет role img изображениям

kattackkattack committed:
Меняет лишние заголовки на aria-label

kattackkattack committed:
Добавляет размеры контентным изображениям

kattackkattack committed:
Причесывает файлы стилей

В соответствии с код гайдом

kattackkattack committed:
Изменяет структуру блоков внутри главной страницы

kattackkattack committed:
Убирает взаимодействие с последней хлебой крошкой

kattackkattack commented:
Из-за использования clip-path в hero обрезается логотип. Починить не получается.

kattackkattack committed:
Добавляет разметку модального окна

Пока что добавила в меню как отдельную страницу, разметку веду в отдельном файле. Когда доделаю — перемещу в index.html
